libraries/bezier-rs/src/bezier/mod.rs

```diff
@@ -0,0 +1,51 @@
+#[cfg(test)]
+pub(super) mod compare;
+
+mod core;
+mod lookup;
+mod manipulators;
+mod solvers;
+mod structs;
+mod transform;
+
+use crate::consts::*;
+use crate::utils;
+pub use structs::*;
```

Comment:
Since this last one is `pub` unlike the previous two, please put a newline above it to separate the non-pub and pub ones.